en/docs/administer/key-managers/configure-auth0-connector.md

```diff
@@ -0,0 +1,302 @@
+# Configure Auth0 as a Key Manager
+
+It is possible to integrate the WSO2 API Manager with an external Identity and Access Management server (IAM) using the Auth0 OAuth Authorization Server to manage the OAuth clients and tokens that are required by WSO2 API Manager. WSO2 API Manager has inbuilt support to consume APIs exposed by Auth0 OAuth.
+
+Follow the instructions below to configure Auth0 as a third-party Key Manager:
+
+!!! info
+    For more information, see the [Getting Started Guide, which is under the official Auth0 documentation](https://auth0.com/docs/get-started).
+
+## Step 1 - Configure Auth0
+
+1.  Create an Auth0 account. Get the URL for the tenant and then sign in to the dashboard.
+
+   [![auth0 dashboard]({{base_path}}/assets/img/administer/auth0-dashboard.png)]({{base_path}}/assets/img/administer/auth0-dashboard.png)
```

Comment:
The indentation is incorrect for this image as a result the auto numbering list has broken and step 2 appears as step 1.